src/App.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/calculator';
 import './index.css';
+import Home from './components/Navlinks/Home';
+import Nav from './components/Navlinks/Nav';
+import Quote from './components/Navlinks/Quote';
 
 // eslint-disable-next-line react/prefer-stateless-function
-class App extends React.Component {
-  render() {
-    return (
-      <div className="App">
-        <section className="app-container" />
-        <Calculator />
-      </div>
-    );
-  }
-}
+const App = () => (
+  <div className="App">
+    <Router>
+      <Nav />
```

Comment:
- [ ] @Graycemuthui great work implementing the changes requested, however, you have not still added the title, which should be on the top left corner as in the initial design as shown below:

 Design Given         |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/177600067-a64812a9-b288-42f5-90d5-f1af7ad62f52.png) |  ![image](https://user-images.githubusercontent.com/61408860/177600190-fa52ccdb-8927-410c-aa8b-14e79a41cb1d.png)